src/main/java/uk/gov/digital/ho/hocs/casework/api/factory/strategies/CopyBFToSMC.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.digital.ho.hocs.casework.api.factory.strategies;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.casework.api.CaseDataService;
+import uk.gov.digital.ho.hocs.casework.api.CorrespondentService;
+import uk.gov.digital.ho.hocs.casework.api.factory.CaseCopy;
+import uk.gov.digital.ho.hocs.casework.domain.model.CaseData;
+
+@Service
+@CaseCopy(fromCaseType = "BF", toCaseType = "SMC")
+public class CopyBFToSMC extends AbstractCaseCopyStrategy implements CaseCopyStrategy {
+
+    private static final String[] DATA_CLOB_KEYS = {
+            "Channel",
+            "CaseSummary",
+            "PrevCompRef",
+            "3rdPartyRef",
+            "OwningCSU",
+            "ComplainantDOB",
+            "ComplainantHORef",
+            "ComplainantPortRef",
+            "ComplainantCompanyName",
+            "ComplainantNationality",
+            "ComplainantGender",
+            "Region",
+            "BusArea",
+            "EnquiryReason1",
```

Comment:
You'll need the most up to date info from BF... the `EnquiryReasonX` fields no longer exist in BF/BF2 and are now called `ComplaintReason...` and have 5 of them, as well as associated `Other` fields. 

That being said, SMC has no concept of these fields so do they need to be copied? You might want to wait for the meeting on the transfer items

Reply:
I shouldn't have added these fields as they come from triage and it's only possible to transfer before triage. I've taken them and any other non-registration fields out